app/templates/summary.html

```diff
@@ -22,13 +22,27 @@ <h2 class="header__subtitle mars">Please check carefully before submission.</h2>
 
       {% for item in block.items %}
 
+      {% for answer in item.state_answers %}
+
         {% set id = summary_id ~ "-" ~ loop.index0 %}
 
-        <dt class="summary__question summary__question--{{item.type}}" id="{{id}}">{{item.question | safe}}</dt>
+        <dt class="summary__question summary__question--{{item.type}}" id="{{id}}">
+          {% if item.state_answers|length == 1 %}
```

Comment:
It's worth taking another look at the changes. I have minimised the logic in the template now although there is still a conditional check due to the dual nature of the date range question/answer